add training DEL page (#20)
ROvACC/docs: training/DELIVERY.md

# DELIVERY CONTROLLER

**  **

Approved by George Enciu, vACC Director

**  **

Created by Teodor G Petrică, OPS

**  **

## 1. Position overview

The Delivery position is the first ATC position that a pilot usually interacts with when preparing to fly. Some airports do not have a dedicated position for Delivery, its work being delegated to the Ground or Tower controller. 

> In the airspace of VATSIM Romania, the only such position is at Henri Coanda International Airport, Bucharest - Otopeni. Although this training is specifically written for the Delivery control position, all this theory can also be used for the Ground or Tower control position, where there is no Delivery or Ground.

Reviewer: ```suggestion
In the airspace of Romania, the only such position is at Henri Coanda International Airport, Bucharest - Otopeni. Although this training is specifically written for the Delivery control position, all this theory can also be used for the Ground or Tower control position, where there is no Delivery or Ground.
```

> The main objective that ATC has from the position of Delivery is to issue Clearances. These are used to give instructions to the pilots immediately after take-off until they reach the en-route system.

Reviewer: ```suggestion
The main objective of the Delivery position is to issue Departure Clearances. These are used to give instructions to the pilots immediately after take-off until they reach the en-route system.
```

> The Delivery controller (abbreviated DEL) is the one who verifies the Flight Plan (abbreviated FPL) - a document that the pilot files in the system so that ATC knows where the pilot intends to navigate their aircraft.

Reviewer: ```suggestion
The Delivery controller (DEL) is the one who verifies the Flight Plan (FPL) - a document that the pilot files with VATSIM so ATC knows where the pilot intends to navigate their aircraft.
```

| Login Call Sign | Call sign | Frequency | ID |
| :-------------: | :-------: | :-------: | :--: |
| LROP_DEL | Otopeni Clearance Delivery | 121.950 | DOP |

## 2. IFR Clearance

> Clearance is the authorization by which the pilot is informed about the point to which he is authorised to fly, the runway for take-off, the route they will follow until the route system, the level or altitude at which they will initially or finally follow and a squawk code.

Reviewer: the point to which he is authorized to fly? adica?

Author: the clearance limit, such as, the final destination (IFR) or the last point where the aircraft is flying IFR( such as a nav aid, 5LNC, etc)

`CLIMB VIA SID FL280`

### 2.5. Transponder Code

The transponder is a device installed in the plane used to identify the aircraft and correlate the target with the flight plan. What you need to know about the transponder is that aircraft are required to use it in controlled airspace and that it can be operated in 3 modes.

Functionality of the transponder: the pilot of the aircraft set a code in the transponder, a code that is transmitted to the radar, the radar identifies a position from where a signal is emitted and is transmitted to the CWP (controller work position), where ATC is tasked with identifying the aircraft that displays a certain code and link or correlate the target on the screen with the flight plan related to that aircraft. As a DEL controller, your job is not to identify the aircraft on the radar because you work in a non-radar environment, but only to instruct the pilot to set a certain transponder code.

There are 4 operating modes of the transponder:

1. STBY (stand-by): transponder is off; no signal is send to the radar
2. ModeA: only the current position is send to the radar
3. ModeC: the current position and altitude is send to the radar

> 4. ModeS: is the most complicated mode, because in addition to ModeC's transmission capability, ModeS can also transmit other data about the aircraft, such as indicated air speed, mach number or fuel on board.

Reviewer: ```suggestion
4. ModeS: is the most complex mode, because in addition to ModeC's transmission capability, ModeS can also transmit other data about the aircraft, such as indicated air speed, mach number or fuel on board.
```
